app/views/profiles/_attendances_details.html.erb

```diff
@@ -11,11 +11,17 @@
     </div>
 
     <ul class="<%= status %> attendees-list">
-      <% attendances.map(&:user).each do |user| %>
+      <% attendances.each do |attendance| %>
+        <% user = attendance.user %>
         <li class="attendee-names">
           <%= link_to user.name, "mailto:#{user.email}", :target => "_blank" %>
+          <% if ShowAttendeePhoneNumberForUser.new(current_user, attendance).call() %>
```

Comment:
you can instantiate instance variables in the controller method that renders this view. Since this is an a partial, you'd have to pass that instance variable into the partial as a "local" also, like @nickbarnwell suggested. Your intuition is right, instantiating inside the view template itself is a little weird. 

the `.call` pattern is mine. I like it for service classes that only have a single public API. Sometimes it sounds dumb though and naming the method something else makes sense, but in general the idea is that you have an object and it does one thing, and you trigger that one thing with a single method. It's just easier if you call that method the same thing.